### PR TITLE
Updated Signup, Login, and Edit Profile devise views

### PR DESCRIPTION
**Pull request by AmandaPerino**
Hi all! I updated the look and feel of the sign up, login, edit profile and other devise views. It could use more styling (not my forté) but for now it is functional. Added css styling to the application.scss file in the end.

Screenshot 2022-11-04 at 17 26 06
Screenshot 2022-11-04 at 17 25 58
Screenshot 2022-11-04 at 17 25 35

### CODE REVIEW

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -1,37 +1,41 @@
...
+                  required: true,
+                  hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length),
+                  input_html: { autocomplete: "new-password" } %>
+      <%= f.input :password_confirmation,
+                  required: true,
+                  input_html: { autocomplete: "new-password" } %>
+    </div>
 
-  <div class="form-actions">
-    <%= f.button :submit, "Sign up" %>
```
Comment (AmandaPerino) [inline app/views/devise/registrations/new.html.erb:33]: Every time I tried to update the buttons with "btn btn-light", it made the buttons too large and pushed them left. Maybe someone knows how I can update the button to be centered and light?

Comment (DariBerrie) [reply app/views/devise/registrations/new.html.erb:33]: Can you share the code you would have used using btn btn-light?

**Comment by DariBerrie** [comment]
I think we can make the background color even lighter, but the text for the labels should be a bit more weight.

When I was going through the sign-up and login, it was not very clear that if someone doesn't have an account, they need to sign up first...So if I just enter my email and password, then click sign up, it takes me to the sign-up page, which is a bit jarring. 

Maybe it should read, "If you don't have an account, sign up here" or something like that so it's clear that this is a separate process.  

What do we think?

**Comment by PiotrekCodes** [comment]
Screenshot 2022-11-05 at 07 34 35
I am working on my personal project as well and this is what I have for register and login screens
Screenshot 2022-11-05 at 07 36 04

File: app/views/devise/registrations/new.html.erb (hunk 1 above)
Comment (AmandaPerino) [reply app/views/devise/registrations/new.html.erb:33]: <div class="form-actions btn btn-light">
    <%= f.button :submit, "Sign up" %>

**Pull request merged**